### ISSUE TITLE
Error:"line 2:8: Column 'db_013318ac9025131b.event_vd.$brand' cannot be resolved"

### ISSUE THREAD

**Issue by larryisthere** [metabase/metabase#10715]
Version:0.33.0.0
Datasource: Presto
Platform: Mac

event_vd is a view.

![image](https://user-images.githubusercontent.com/271518/63699894-5297d100-c854-11e9-971c-03a9b5e9332c.png)

**Comment by larryisthere** [comment]
@camsaul no, it was a data browse. BTW, the write SQL worked well.

**Comment by larryisthere** [comment]
Thank you @flamber. And there was no related error about presto and syncing. But I've got details of the above error.

```
[d43fd207-ad22-4bf8-af3e-543bdaff235c] 2019-09-03T14:50:35+08:00 WARN metabase.query-processor.middleware.process-userland-query Query failure {:status :failed, :class clojure.lang.ExceptionInfo, :error "line 2:8: Column 'db_1390517108757668u.event_vd.$app_version' cannot be resolved", :stacktrace ("--> driver.presto$execute_presto_query_BANG_$fn__476.invoke(presto.clj:131)" "util.ssh$with_ssh_tunnel_STAR_.invokeStatic(ssh.clj:106)" "util.ssh$with_ssh_tunnel_STAR_.invoke(ssh.clj:91)" "driver.presto$execute_presto_query_BANG_.invokeStatic(presto.clj:125)" "driver.presto$execute_presto_query_BANG_.invoke(presto.clj:121)" "driver.presto$eval671$fn__673.invoke(presto.clj:272)" "query_processor$fn__43931$execute_query__43936$fn__43937.invoke(query_processor.clj:70)" "query_processor$fn__43931$execute_query__43936.invoke(query_processor.clj:64)" "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__34065.invoke(mbql_to_native.clj:38)" "query_processor.middleware.annotate$result_rows_maps__GT_vectors$fn__36309.invoke(annotate.clj:540)" "query_processor.middleware.annotate$add_column_info$fn__36215.invoke(annotate.clj:484)" "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__37236.invoke(cumulative_aggregations.clj:57)" "query_processor.middleware.resolve_joins$resolve_joins$fn__41034.invoke(resolve_joins.clj:184)" "query_processor.middleware.limit$limit$fn__37972.invoke(limit.clj:19)" "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__43800.invoke(results_metadata.clj:87)" "query_processor.middleware.format_rows$format_rows$fn__37960.invoke(format_rows.clj:26)" "query_processor.middleware.add_dimension_projections$add_remapping$fn__34824.invoke(add_dimension_projections.clj:234)" "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__35443.invoke(add_source_metadata.clj:107)" "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__41084.invoke(resolve_source_table.clj:46)" "query_processor.middleware.add_row_count_and_status$add_row_count_and_status$fn__35303.invoke(add_row_count_and_status.clj:16)" "query_processor.middleware.driver_specific$process_query_in_context$fn__37447.invoke(driver_specific.clj:12)" "query_processor.middleware.resolve_driver$resolve_driver$fn__40698.invoke(resolve_driver.clj:22)" "query_processor.middleware.bind_effective_timezone$bind_effective_timezone$fn__36638$fn__36639.invoke(bind_effective_timezone.clj:9)" "util.date$call_with_effective_timezone.invokeStatic(date.clj:88)" "util.date$call_with_effective_timezone.invoke(date.clj:77)" "query_processor.middleware.bind_effective_timezone$bind_effective_timezone$fn__36638.invoke(bind_effective_timezone.clj:8)" "query_processor.middleware.store$initialize_store$fn__43825$fn__43826.invoke(store.clj:11)" "query_processor.store$do_with_store.invokeStatic(store.clj:46)" "query_processor.store$do_with_store.invoke(store.clj:40)" "query_processor.middleware.store$initialize_store$fn__43825.invoke(store.clj:10)" "query_processor.middleware.async$async__GT_sync$fn__33976.invoke(async.clj:23)" "query_processor.middleware.async_wait$runnable$fn__36366.invoke(async_wait.clj:89)"), :query {:type "query", :query {:source-table 43}, :parameters [], :async? true, :middleware {:add-default-userland-constraints? true, :userland-query? true}, :info {:executed-by 1, :context :ad-hoc, :card-id nil, :nested? false, :query-hash [-38, 83, -114, 87, 26, -122, 106, -71, -16, -127, 103, -81, 103, -70, 20, 9, 76, 102, 110, -106, 71, 61, 70, -52, -68, 105, -39, -46, 72, -83, 23, 46]}, :constraints {:max-results 10000, :max-results-bare-rows 2000}}, :preprocessed {:constraints {:max-results 10000, :max-results-bare-rows 2000}, :type :query, :middleware {:add-default-userland-constraints? true, :userland-query? true}, :info {:executed-by 1, :context :ad-hoc, :nested? false, :query-hash [-38, 83, -114, 87, 26, -122, 106, -71, -16, -127, 103, -81, 103, -70, 20, 9, 76, 102, 110, -106, 71, 61, 70, -52, -68, 105, -39, -46, 72, -83, 23, 46]}, :preprocessing-level 1, :database 2, :driver :presto, :query {:source-table 43, :fields [[:field-id 4008] [:field-id 531] [:field-id 567] [:field-id 628] [:field-id 601] [:field-id 544] [:field-id 503] [:field-id 483] [:field-id 598] [:field-id 632] [:field-id 523] [:field-id 4016] [:field-id 619] [:field-id 553] [:field-id 539] [:field-id 609] [:field-id 477] [:field-id 535] [:field-id 618] [:field-id 574] [:field-id 549] [:field-id 482] [:field-id 579] [:field-id 566] [:field-id 597] [:field-id 4015] [:field-id 611] [:field-id 573] [:field-id 581] [:field-id 545] [:field-id 613] [:field-id 4017] [:field-id 493] [:field-id 4010] [:field-id 584] [:field-id 568] [:field-id 518] [:field-id 624] [:field-id 542] [:field-id 587] [:field-id 4012] [:field-id 4009] [:field-id 4018] [:field-id 4014] [:field-id 4013] [:field-id 562] [:field-id 536] [:field-id 595] [:field-id 515] [:field-id 610] [:field-id 569] [:field-id 506] [:field-id 582] [:field-id 510] [:field-id 519] [:field-id 586] [:field-id 494] [:field-id 528] [:field-id 615] [:field-id 630] [:field-id 593] [:field-id 530] [:field-id 578] [:field-id 541] [:field-id 502] [:field-id 631] [:field-id 575] [:field-id 602] [:field-id 529] [:field-id 540] [:field-id 623] [:field-id 527] [:field-id 556] [:field-id 478] [:field-id 492] [:field-id 600] [:field-id 511] [:field-id 555] [:field-id 488] [:field-id 486] [:field-id 604] [:field-id 512] [:field-id 607] [:field-id 497] [:field-id 4011] [:field-id 524] [:field-id 514] [:field-id 590] [:field-id 554] [:field-id 526] [:field-id 605] [:field-id 517] [:field-id 577] [:field-id 596] [:field-id 625] [:field-id 489] [:field-id 558] [:field-id 547] [:field-id 620] [:field-id 560] [:field-id 585] [:field-id 592] [:field-id 551] [:field-id 563] [:field-id 583] [:field-id 480] [:field-id 532] [:field-id 614] [:field-id 606] [:field-id 509] [:field-id 603] [:field-id 499] [:field-id 557] [:field-id 520] [:field-id 484] [:field-id 565] [:field-id 626] [:field-id 594] [:field-id 546] [:field-id 498] [:field-id 496] [:field-id 591] [:field-id 571] [:field-id 570] [:field-id 550] [:field-id 621] [:field-id 564] [:field-id 548] [:field-id 479] [:field-id 490] [:field-id 522] [:field-id 608] [:field-id 537] [:field-id 500] [:field-id 576] [:field-id 487] [:field-id 534] [:field-id 559] [:field-id 572] [:field-id 508] [:field-id 538] [:field-id 507] [:field-id 589] [:field-id 617] [:field-id 481] [:field-id 516] [:field-id 525] [:field-id 521] [:field-id 504] [:field-id 616] [:field-id 622] [:field-id 505] [:field-id 627] [:field-id 491] [:field-id 588] [:field-id 552] [:field-id 485] [:field-id 501] [:field-id 612] [:field-id 543] [:field-id 495] [:field-id 561] [:field-id 580] [:field-id 513] [:field-id 599] [:field-id 629] [:field-id 533]], :limit 2000}}, :native {:query "SELECT \"db_1390517108757668u\".\"event_vd\".\"$app_version\" AS \"$app_version\", \"db_1390517108757668u\".\"event_vd\".\"$brand\" AS \"$brand\", \"db_1390517108757668u\".\"event_vd\".\"$browser\" AS \"$browser\", \"db_1390517108757668u\".\"event_vd\".\"$browser_version\" AS \"$browser_version\", \"db_1390517108757668u\".\"event_vd\".\"$campaign_shortlink\" AS \"$campaign_shortlink\", \"db_1390517108757668u\".\"event_vd\".\"$channel\" AS \"$channel\", \"db_1390517108757668u\".\"event_vd\".\"$city\" AS \"$city\", \"db_1390517108757668u\".\"event_vd\".\"$click_x\" AS \"$click_x\", \"db_1390517108757668u\".\"event_vd\".\"$click_y\" AS \"$click_y\", \"db_1390517108757668u\".\"event_vd\".\"$country\" AS \"$country\", \"db_1390517108757668u\".\"event_vd\".\"$device_type\" AS \"$device_type\", \"db_1390517108757668u\".\"event_vd\".\"$duration\" AS \"$duration\", \"db_1390517108757668u\".\"event_vd\".\"$element_clickable\" AS \"$element_clickable\", \"db_1390517108757668u\".\"event_vd\".\"$element_content\" AS \"$element_content\", \"db_1390517108757668u\".\"event_vd\".\"$element_path\" AS \"$element_path\", \"db_1390517108757668u\".\"event_vd\".\"$element_type\" AS \"$element_type\", \"db_1390517108757668u\".\"event_vd\".\"$element_x\" AS \"$element_x\", \"db_1390517108757668u\".\"event_vd\".\"$element_y\" AS \"$element_y\", \"db_1390517108757668u\".\"event_vd\".\"$errormsg\" AS \"$errormsg\", \"db_1390517108757668u\".\"event_vd\".\"$event_duration\" AS \"$event_duration\", \"db_1390517108757668u\".\"event_vd\".\"$fingerprint\" AS \"$fingerprint\", \"db_1390517108757668u\".\"event_vd\".\"$incognito_mode\" AS \"$incognito_mode\", \"db_1390517108757668u\".\"event_vd\".\"$ip\" AS \"$ip\", \"db_1390517108757668u\".\"event_vd\".\"$is_first_day\" AS \"$is_first_day\", \"db_1390517108757668u\".\"event_vd\".\"$is_first_time\" AS \"$is_first_time\", \"db_1390517108757668u\".\"event_vd\".\"$is_from_background\" AS \"$is_from_background\", \"db_1390517108757668u\".\"event_vd\".\"$is_login\" AS \"$is_login\", \"db_1390517108757668u\".\"event_vd\".\"$is_time_calibrated\" AS \"$is_time_calibrated\", \"db_1390517108757668u\".\"event_vd\".\"$language\" AS \"$language\", \"db_1390517108757668u\".\"event_vd\".\"$lib\" AS \"$lib\", \"db_1390517108757668u\".\"event_vd\".\"$lib_version\" AS \"$lib_version\", \"db_1390517108757668u\".\"event_vd\".\"$manufacturer\" AS \"$manufacturer\", \"db_1390517108757668u\".\"event_vd\".\"$model\" AS \"$model\", \"db_1390517108757668u\".\"event_vd\".\"$network\" AS \"$network\", \"db_1390517108757668u\".\"event_vd\".\"$os\" AS \"$os\", \"db_1390517108757668u\".\"event_vd\".\"$os_version\" AS \"$os_version\", \"db_1390517108757668u\".\"event_vd\".\"$page_height\" AS \"$page_height\", \"db_1390517108757668u\".\"event_vd\".\"$page_width\" AS \"$page_width\", \"db_1390517108757668u\".\"event_vd\".\"$pagename\" AS \"$pagename\", \"db_1390517108757668u\".\"event_vd\".\"$platform\" AS \"$platform\", \"db_1390517108757668u\".\"event_vd\".\"$prop_0\" AS \"$prop_0\", \"db_1390517108757668u\".\"event_vd\".\"$prop_1\" AS \"$prop_1\", \"db_1390517108757668u\".\"event_vd\".\"$prop_2\" AS \"$prop_2\", \"db_1390517108757668u\".\"event_vd\".\"$prop_3\" AS \"$prop_3\", \"db_1390517108757668u\".\"event_vd\".\"$prop_4\" AS \"$prop_4\", \"db_1390517108757668u\".\"event_vd\".\"$province\" AS \"$province\", \"db_1390517108757668u\".\"event_vd\".\"$referrer\" AS \"$referrer\", \"db_1390517108757668u\".\"event_vd\".\"$referrer_domain\" AS \"$referrer_domain\", \"db_1390517108757668u\".\"event_vd\".\"$scene\" AS \"$scene\", \"db_1390517108757668u\".\"event_vd\".\"$scene_type\" AS \"$scene_type\", \"db_1390517108757668u\".\"event_vd\".\"$screen_height\" AS \"$screen_height\", \"db_1390517108757668u\".\"event_vd\".\"$screen_width\" AS \"$screen_width\", \"db_1390517108757668u\".\"event_vd\".\"$search_engine\" AS \"$search_engine\", \"db_1390517108757668u\".\"event_vd\".\"$search_keyword\" AS \"$search_keyword\", \"db_1390517108757668u\".\"event_vd\".\"$search_keywords\" AS \"$search_keywords\", \"db_1390517108757668u\".\"event_vd\".\"$session_id\" AS \"$session_id\", \"db_1390517108757668u\".\"event_vd\".\"$signup_time\" AS \"$signup_time\", \"db_1390517108757668u\".\"event_vd\".\"$social\" AS \"$social\", \"db_1390517108757668u\".\"event_vd\".\"$social_media\" AS \"$social_media\", \"db_1390517108757668u\".\"event_vd\".\"$social_share_from\" AS \"$social_share_from\", \"db_1390517108757668u\".\"event_vd\".\"$startup_time\" AS \"$startup_time\", \"db_1390517108757668u\".\"event_vd\".\"$time_zone\" AS \"$time_zone\", \"db_1390517108757668u\".\"event_vd\".\"$title\" AS \"$title\", \"db_1390517108757668u\".\"event_vd\".\"$track_xwhen\" AS \"$track_xwhen\", \"db_1390517108757668u\".\"event_vd\".\"$traffic_source_type\" AS \"$traffic_source_type\", \"db_1390517108757668u\".\"event_vd\".\"$url\" AS \"$url\", \"db_1390517108757668u\".\"event_vd\".\"$url_domain\" AS \"$url_domain\", \"db_1390517108757668u\".\"event_vd\".\"$url_path\" AS \"$url_path\", \"db_1390517108757668u\".\"event_vd\".\"$user_agent\" AS \"$user_agent\", \"db_1390517108757668u\".\"event_vd\".\"$utm_campaign\" AS \"$utm_campaign\", \"db_1390517108757668u\".\"event_vd\".\"$utm_campaign_id\" AS \"$utm_campaign_id\", \"db_1390517108757668u\".\"event_vd\".\"$utm_content\" AS \"$utm_content\", \"db_1390517108757668u\".\"event_vd\".\"$utm_media\" AS \"$utm_media\", \"db_1390517108757668u\".\"event_vd\".\"$utm_medium\" AS \"$utm_medium\", \"db_1390517108757668u\".\"event_vd\".\"$utm_source\" AS \"$utm_source\", \"db_1390517108757668u\".\"event_vd\".\"$utm_term\" AS \"$utm_term\", \"db_1390517108757668u\".\"event_vd\".\"$viewport_height\" AS \"$viewport_height\", \"db_1390517108757668u\".\"event_vd\".\"$viewport_position\" AS \"$viewport_position\", \"db_1390517108757668u\".\"event_vd\".\"$viewport_width\" AS \"$viewport_width\", \"db_1390517108757668u\".\"event_vd\".\"$web_crawler\" AS \"$web_crawler\", \"db_1390517108757668u\".\"event_vd\".\"aa\" AS \"aa\", \"db_1390517108757668u\".\"event_vd\".\"analysis_code\" AS \"analysis_code\", \"db_1390517108757668u\".\"event_vd\".\"analysisname\" AS \"analysisname\", \"db_1390517108757668u\".\"event_vd\".\"board_name\" AS \"board_name\", \"db_1390517108757668u\".\"event_vd\".\"board_type\" AS \"board_type\", \"db_1390517108757668u\".\"event_vd\".\"boardname\" AS \"boardname\", \"db_1390517108757668u\".\"event_vd\".\"categoryid\" AS \"categoryid\", \"db_1390517108757668u\".\"event_vd\".\"ccc\" AS \"ccc\", \"db_1390517108757668u\".\"event_vd\".\"chart_count\" AS \"chart_count\", \"db_1390517108757668u\".\"event_vd\".\"chart_name\" AS \"chart_name\", \"db_1390517108757668u\".\"event_vd\".\"chart_type\" AS \"chart_type\", \"db_1390517108757668u\".\"event_vd\".\"click_position\" AS \"click_position\", \"db_1390517108757668u\".\"event_vd\".\"cohort_name\" AS \"cohort_name\", \"db_1390517108757668u\".\"event_vd\".\"cohort_type\" AS \"cohort_type\", \"db_1390517108757668u\".\"event_vd\".\"cohortname\" AS \"cohortname\", \"db_1390517108757668u\".\"event_vd\".\"cohorts\" AS \"cohorts\", \"db_1390517108757668u\".\"event_vd\".\"company\" AS \"company\", \"db_1390517108757668u\".\"event_vd\".\"company_code\" AS \"company_code\", \"db_1390517108757668u\".\"event_vd\".\"company_size\" AS \"company_size\", \"db_1390517108757668u\".\"event_vd\".\"condition_type\" AS \"condition_type\", \"db_1390517108757668u\".\"event_vd\".\"conditions\" AS \"conditions\", \"db_1390517108757668u\".\"event_vd\".\"corporate_name\" AS \"corporate_name\", \"db_1390517108757668u\".\"event_vd\".\"count\" AS \"count\", \"db_1390517108757668u\".\"event_vd\".\"creation_time\" AS \"creation_time\", \"db_1390517108757668u\".\"event_vd\".\"datafrom\" AS \"datafrom\", \"db_1390517108757668u\".\"event_vd\".\"demo_type\" AS \"demo_type\", \"db_1390517108757668u\".\"event_vd\".\"department\" AS \"department\", \"db_1390517108757668u\".\"event_vd\".\"dimensions\" AS \"dimensions\", \"db_1390517108757668u\".\"event_vd\".\"distinct_id\" AS \"distinct_id\", \"db_1390517108757668u\".\"event_vd\".\"domain\" AS \"domain\", \"db_1390517108757668u\".\"event_vd\".\"drill_type\" AS \"drill_type\", \"db_1390517108757668u\".\"event_vd\".\"ds\" AS \"ds\", \"db_1390517108757668u\".\"event_vd\".\"email\" AS \"email\", \"db_1390517108757668u\".\"event_vd\".\"events\" AS \"events\", \"db_1390517108757668u\".\"event_vd\".\"failurereason\" AS \"failurereason\", \"db_1390517108757668u\".\"event_vd\".\"fieldname\" AS \"fieldname\", \"db_1390517108757668u\".\"event_vd\".\"filters\" AS \"filters\", \"db_1390517108757668u\".\"event_vd\".\"founder\" AS \"founder\", \"db_1390517108757668u\".\"event_vd\".\"groups\" AS \"groups\", \"db_1390517108757668u\".\"event_vd\".\"help_type\" AS \"help_type\", \"db_1390517108757668u\".\"event_vd\".\"hrk\" AS \"hrk\", \"db_1390517108757668u\".\"event_vd\".\"is_attributes\" AS \"is_attributes\", \"db_1390517108757668u\".\"event_vd\".\"is_logo\" AS \"is_logo\", \"db_1390517108757668u\".\"event_vd\".\"isagree\" AS \"isagree\", \"db_1390517108757668u\".\"event_vd\".\"logical\" AS \"logical\", \"db_1390517108757668u\".\"event_vd\".\"mail\" AS \"mail\", \"db_1390517108757668u\".\"event_vd\".\"method\" AS \"method\", \"db_1390517108757668u\".\"event_vd\".\"metrics\" AS \"metrics\", \"db_1390517108757668u\".\"event_vd\".\"metrics_type\" AS \"metrics_type\", \"db_1390517108757668u\".\"event_vd\".\"name\" AS \"name\", \"db_1390517108757668u\".\"event_vd\".\"nav_name\" AS \"nav_name\", \"db_1390517108757668u\".\"event_vd\".\"navname\" AS \"navname\", \"db_1390517108757668u\".\"event_vd\".\"nps_comment\" AS \"nps_comment\", \"db_1390517108757668u\".\"event_vd\".\"nps_score\" AS \"nps_score\", \"db_1390517108757668u\".\"event_vd\".\"panel_name\" AS \"panel_name\", \"db_1390517108757668u\".\"event_vd\".\"param\" AS \"param\", \"db_1390517108757668u\".\"event_vd\".\"phone\" AS \"phone\", \"db_1390517108757668u\".\"event_vd\".\"position\" AS \"position\", \"db_1390517108757668u\".\"event_vd\".\"position_cohorts_new\" AS \"position_cohorts_new\", \"db_1390517108757668u\".\"event_vd\".\"position_copy\" AS \"position_copy\", \"db_1390517108757668u\".\"event_vd\".\"position_dl\" AS \"position_dl\", \"db_1390517108757668u\".\"event_vd\".\"position_email_new\" AS \"position_email_new\", \"db_1390517108757668u\".\"event_vd\".\"position_engage_cohorts\" AS \"position_engage_cohorts\", \"db_1390517108757668u\".\"event_vd\".\"position_page\" AS \"position_page\", \"db_1390517108757668u\".\"event_vd\".\"position_push_new\" AS \"position_push_new\", \"db_1390517108757668u\".\"event_vd\".\"position_sms_new\" AS \"position_sms_new\", \"db_1390517108757668u\".\"event_vd\".\"productid\" AS \"productid\", \"db_1390517108757668u\".\"event_vd\".\"project_name\" AS \"project_name\", \"db_1390517108757668u\".\"event_vd\".\"properties\" AS \"properties\", \"db_1390517108757668u\".\"event_vd\".\"property_name\" AS \"property_name\", \"db_1390517108757668u\".\"event_vd\".\"query\" AS \"query\", \"db_1390517108757668u\".\"event_vd\".\"referrer_demo\" AS \"referrer_demo\", \"db_1390517108757668u\".\"event_vd\".\"region\" AS \"region\", \"db_1390517108757668u\".\"event_vd\".\"root_type\" AS \"root_type\", \"db_1390517108757668u\".\"event_vd\".\"scene_type\" AS \"scene_type\", \"db_1390517108757668u\".\"event_vd\".\"signdate\" AS \"signdate\", \"db_1390517108757668u\".\"event_vd\".\"size\" AS \"size\", \"db_1390517108757668u\".\"event_vd\".\"sourcetype\" AS \"sourcetype\", \"db_1390517108757668u\".\"event_vd\".\"template_name\" AS \"template_name\", \"db_1390517108757668u\".\"event_vd\".\"time_type\" AS \"time_type\", \"db_1390517108757668u\".\"event_vd\".\"topic\" AS \"topic\", \"db_1390517108757668u\".\"event_vd\".\"type\" AS \"type\", \"db_1390517108757668u\".\"event_vd\".\"username\" AS \"username\", \"db_1390517108757668u\".\"event_vd\".\"usertype\" AS \"usertype\", \"db_1390517108757668u\".\"event_vd\".\"xwhat\" AS \"xwhat\", \"db_1390517108757668u\".\"event_vd\".\"xwhen\" AS \"xwhen\", \"db_1390517108757668u\".\"event_vd\".\"xwho\" AS \"xwho\" FROM \"db_1390517108757668u\".\"event_vd\" LIMIT 2000", :params nil}, :ex-data {:message "line 2:8: Column 'db_1390517108757668u.event_vd.$app_version' cannot be resolved", :errorCode 1, :errorName "SYNTAX_ERROR", :errorType "USER_ERROR", :errorLocation {:lineNumber 2, :columnNumber 8}, :failureInfo {:type "com.facebook.presto.sql.analyzer.SemanticException", :message "line 2:8: Column 'db_1390517108757668u.event_vd.$app_version' cannot be resolved", :suppressed [], :stack ["com.facebook.presto.sql.analyzer.SemanticExceptions.missingAttributeException(SemanticExceptions.java:30)" "com.facebook.presto.sql.analyzer.ExpressionAnalyzer$Visitor.visitDereferenceExpression(ExpressionAnalyzer.java:413)" "com.facebook.presto.sql.analyzer.ExpressionAnalyzer$Visitor.visitDereferenceExpression(ExpressionAnalyzer.java:293)" "com.facebook.presto.sql.tree.DereferenceExpression.accept(DereferenceExpression.java:54)" "com.facebook.presto.sql.tree.StackableAstVisitor.process(StackableAstVisitor.java:26)" "com.facebook.presto.sql.analyzer.ExpressionAnalyzer$Visitor.process(ExpressionAnalyzer.java:314)" "com.facebook.presto.sql.analyzer.ExpressionAnalyzer.analyze(ExpressionAnalyzer.java:264)" "com.facebook.presto.sql.analyzer.ExpressionAnalyzer.analyzeExpression(ExpressionAnalyzer.java:1537)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.analyzeExpression(StatementAnalyzer.java:2010)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.analyzeSelect(StatementAnalyzer.java:1821)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:946)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:253)" "com.facebook.presto.sql.tree.QuerySpecification.accept(QuerySpecification.java:127)" "com.facebook.presto.sql.tree.AstVisitor.process(AstVisitor.java:27)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:265)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:275)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:671)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:253)" "com.facebook.presto.sql.tree.Query.accept(Query.java:94)" "com.facebook.presto.sql.tree.AstVisitor.process(AstVisitor.java:27)" "com.facebook.presto.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:265)" "com.facebook.presto.sql.analyzer.StatementAnalyzer.analyze(StatementAnalyzer.java:239)" "com.facebook.presto.sql.analyzer.Analyzer.analyze(Analyzer.java:72)" "com.facebook.presto.sql.analyzer.Analyzer.analyze(Analyzer.java:64)" "com.facebook.presto.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:169)" "com.facebook.presto.execution.SqlQueryExecution$SqlQueryExecutionFactory.createQueryExecution(SqlQueryExecution.java:660)" "com.facebook.presto.execution.SqlQueryExecution$SqlQueryExecutionFactory.createQueryExecution(SqlQueryExecution.java:582)" "com.facebook.presto.execution.SqlQueryManager.createQuery(SqlQueryManager.java:417)" "com.facebook.presto.server.protocol.Query.<init>(Query.java:185)" "com.facebook.presto.server.protocol.Query.create(Query.java:152)" "com.facebook.presto.server.protocol.StatementResource.createQuery(StatementResource.java:138)" "sun.reflect.GeneratedMethodAccessor1560.invoke(Unknown Source)" "sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)" "java.lang.reflect.Method.invoke(Method.java:498)" "org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:76)" "org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:148)" "org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:191)" "org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$VoidOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:183)" "org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:103)" "org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:493)" "org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:415)" "org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:104)" "org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:277)" "org.glassfish.jersey.internal.Errors$1.call(Errors.java:272)" "org.glassfish.jersey.internal.Errors$1.call(Errors.java:268)" "org.glassfish.jersey.internal.Errors.process(Errors.java:316)" "org.glassfish.jersey.internal.Errors.process(Errors.java:298)" "org.glassfish.jersey.internal.Errors.process(Errors.java:268)" "org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:289)" "org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:256)" "org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:703)" "org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:416)" "org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:370)" "org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:389)" "org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:342)" "org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:229)" "org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:865)" "org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1655)" "com.facebook.presto.server.security.AuthenticationFilter.doFilter(AuthenticationFilter.java:69)" "org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1642)" "io.airlift.http.server.TraceTokenFilter.doFilter(TraceTokenFilter.java:64)" "org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1642)" "io.airlift.http.server.TimingFilter.doFilter(TimingFilter.java:52)" "org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1642)" "org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533)" "org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:146)" "org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:674)" "org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)" "org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)" "org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)" "org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203)" "org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473)" "org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201)" "org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155)" "org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)" "org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:126)" "org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:169)" "org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:61)" "org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)" "org.eclipse.jetty.server.Server.handle(Server.java:531)" "org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:352)" "org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)" "org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:281)" "org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:102)" "org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)" "org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)" "org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)" "org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)" "org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)" "org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)" "org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:755)" "org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:673)" "java.lang.Thread.run(Thread.java:748)"], :errorLocation {:lineNumber 2, :columnNumber 8}}}}
```

**Comment by flamber** [comment]
Most likely fixed by #12815 available since 0.36.0, which fixed support for special characters in column names like `$test` or `%test`
